myFirstPromise.catch((errorMessage) => {
  console.error("Uh oh! " + errorMessage);
});